README.md

```diff
@@ -217,13 +217,15 @@ ios/tmp.xcconfig
 4. go to project settings
 5. apply config to your configurations
    ![img](./readme-pics/3.ios_apply_config.png)
-6. create new build phase for the scheme which will generate "tmp.xcconfig" before each build exposing values to Build Settings and Info.plist (this snippet has to be placed after "cp ... ${PROJECT_DIR}/../.env" if [approach explained below](#ios-multi-scheme) is used)
+6. Go to _Edit scheme..._ -> _Build_ -> _Pre-actions_, click _+_ and select _New Run Script Action_. Paste below code which will generate "tmp.xcconfig" before each build exposing values to Build Settings and Info.plist. Make sure to select your target under _Provide build settings from_, so `$SRCROOT` environment variables is available to the script. (Note that this snippet has to be placed after "cp ... \${PROJECT_DIR}/../.env" if [approach explained below](#ios-multi-scheme) is used).
 
-```
-"${SRCROOT}/../node_modules/react-native-config/ios/ReactNativeConfig/BuildXCConfig.rb" "${SRCROOT}/.." "${SRCROOT}/tmp.xcconfig"
-```
+   ```
+   "${SRCROOT}/../node_modules/react-native-config/ios/ReactNativeConfig/BuildXCConfig.rb" "${SRCROOT}/.." "${SRCROOT}/tmp.xcconfig"
+   ```
+
+   ![img](./readme-pics/4.ios_pre_actions.png)
 
-7. **Accessing your variable in info.plist**: You can now access your env variable using the prefix `RNC_`, for example `$(RNC_MY_ENV_VARIABLE)`. If you face issues accessing variables, please refer to [this issue](https://github.com/luggit/react-native-config/issues/391#issuecomment-632331803).
+7. You can now access your env variables in the Info.plist, for example `$(MY_ENV_VARIABLE)`. If you face issues accessing variables, please open a new issue and provide as much details as possible so above steps can be improved.
```

Comment:
This seems accurate to me. I was wondering why the variables are always empty when using `$(RNC_MY_ENV_VARIABLE)`. It looks like we don't need the `RNC_` prefix, it should just be `$(MY_ENV_VARIABLE)`.

Reply:
Not sure though if this was intended. I feel like having the prefix would be a good idea to avoid accidentally overwriting other existing variables like `$(PRODUCT_NAME)`, `$(PRODUCT_BUNDLE_IDENTIFIER)`, ...

Reply:
I agree that `RNC_` prefix is useful, but that would be a breaking change.
Maybe someone can make a separate PR to generate both styles of variables and document using `RNC_`-prefixed ones, so it is easier to remove unprefixed variables in the next major.

In any case it is more helpful to describe actual behavior (even undesired) than mislead people :)

Reply:
I agree on that. I was just wondering if the variables had the prefix before, or it was just an incorrect statement in README. I updated recently this library in a project from an older version (`0.11.7`) where the variables were being accesed in Info.plist with `__RN_CONFIG_MY_ENV_VARIABLE` to the latest version (`1.3.3`), so I'm not sure if there was some versions in between that were using the `RNC_` prefix.

Reply:
AFAICT https://github.com/luggit/react-native-config/commit/1eb6ac01991210ddad2989857359a0f6ee35d734 removed `__RN_CONFIG_` prefix and introduced new approach with `.xcconfig` file and then https://github.com/luggit/react-native-config/commit/bc3ca9d261de31e5c79fc9eab284f563078d0511 introduced `RNC_` prefix instead if you use previous approach with `Info.plist` postprocessing, which is no longer documented.

It's messy 🤷

Reply:
And I guess the reason why older approach was restored at the first place is because `.xccconfig` was not misunderstood/not working properly: https://github.com/luggit/react-native-config/issues/391#issuecomment-632331803.